setup.py

```diff
@@ -67,41 +66,43 @@
 ]
 
 install_requires = [
-    'Flask-BabelEx>=0.9.3',
-    'Flask-Breadcrumbs>=0.4.0',
-    'Flask-CeleryExt>=0.2.2',
+    # OD adapted Invenio
```

Comment:
Regarding OD, we have COD3 and COD in the code base already. It might be good to harmonise this, e.g. just say COD3 here. 

However, in the series of  "OD adapted..." comments,  we could simply skip using acronyms, since the context is clear, we are building Invenio web app. So we could simply say "Custom Invenio `files` bundle`" and stuff.  (otherwise we'd have to explain how "adapted" the modules are, isnce most of them we are use vanilla, I think. Only a few required adaptations.)

Just a cosmetic naming comment.